### DISCUSSION TITLE
All the colours have disappeared from videos since installing latest update

### DISCUSSION THREAD

**Discussion by jimmyrustles** [TeamNewPipe/NewPipe#10729] [Questions] [answered]
![Screenshot_20240103_091842_NewPipe](https://github.com/TeamNewPipe/NewPipe/assets/5096684/7332d514-e5e0-41b8-9970-b5d645f1c611) 
This has been happening for quite a while, the videos play fine, it's just the video pictures are in greyscale before selecting them to play, looks like a loading problem? Thanks.

**Comment by SameenAhnaf** [comment, marked as answer]
Try going to Settings > Content > Image Quality > Choose "High quality". If images are still not loading, maybe selecting low or medium quality will help.
![Screenshot_20240103-173740](https://github.com/TeamNewPipe/NewPipe/assets/79245641/752a21c6-efa0-44c9-bb6e-c629d6e3a63b)

**Comment by jimmyrustles** [comment]
Hey,  that worked, thanks!
